mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxExtensions.kt

```diff
@@ -29,6 +29,40 @@ inline fun <T, reified VM : BaseMvRxViewModel<S>, reified S : MvRxState> T.fragm
         .apply { subscribe(this@fragmentViewModel, subscriber = { postInvalidate() }) }
 }
 
+/**
+ * Gets or creates a ViewModel scoped to a parent fragment. This delegate will walk up the parentFragment hierarchy
+ * until it finds a Fragment that can provide the correct ViewModel. If no parent fragments can provide the ViewModel,
+ * a new one will be created in the direct parent of the curent Fragment.
+ */
+inline fun <T, reified VM : BaseMvRxViewModel<S>, reified S : MvRxState> T.parentFragmentViewModel(
+    viewModelClass: KClass<VM> = VM::class,
+    crossinline keyFactory: () -> String = { viewModelClass.java.name }
+): Lazy<VM> where T : Fragment, T : MvRxView = lifecycleAwareLazy(this) {
+    requireNotNull(parentFragment) { "There is no parent fragment!" }
+    val factory = MvRxFactory { error("No ViewModel for this Fragment.") }
+    var fragment: Fragment? = this
```

Comment:
I think it should be `var fragment: Fragment? = parentFragment`, or it will always throw `"There is no parent fragment!"` in the first time.